low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -223,8 +223,8 @@ def validate(self):
         pass
```

Comment:
Please add validation on the args.
1. if no args
2. if invalid args

Reply:
Yes. Added the validators.